psd-web/app/models/user.rb

```diff
@@ -17,21 +17,21 @@ def self.activated
     where(account_activated: true)
   end
 
-  def self.create_and_send_invite(email_address, team, redirect_url)
-    KeycloakClient.instance.create_user(email_address)
+  def self.create_and_send_invite(email_address, team, inviting_user)
+    user = create(
```

Comment:
you want to kown if this fails - if user is invalid what happens?
```suggestion
    user = create!(
```